### PR TITLE
Feature/auth

### PR DESCRIPTION
**Pull request by yonatandt**

### CODE REVIEW

**Comment by omerzamir** [comment]
[This comment](https://github.com/meateam/kd-helm/issues/10#issuecomment-493765390) is relevant here too.

I'll add other comments and change requests later. But first merge your branch with develop, it's outdated.

File: authenticator/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  annotations:
+{{ toYaml . | indent 4 }}
+{{- end }}     
+spec:
+  rules:
+  {{- range .Values.global.ingress.hosts }}
+    - host: {{ . | quote }}
+      http:
+        paths:
+          - path: /auth/?(.*)
```
Comment (omerzamir) [inline authenticator/templates/ingress.yaml]: Regex isn't necessary here.

**Review by omerzamir — CHANGES_REQUESTED** [review]
Beside the comment on the regex, you should change the name of the authenticator repo to authentication-service. 
also, requirement.lock should be ignored with gitignore, it appears in the PR(that is the reason i asked for merge with develop). we should check that it's really ignored.

File: authentication-service/values.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Default values for server-chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  tag: "a15cb95"
```
Comment (omerzamir) [inline authentication-service/values.yaml]: wrong tag.
should be `tag: "3483326"`

**Review by omerzamir — DISMISSED** [review]

**Review by omerzamir — CHANGES_REQUESTED** [review]
see my comment on the tag

**Review by omerzamir — APPROVED** [review]

**Comment by yonatandt** [comment]
Why isn't the branch deleted after merge?

**Pull request merged**